turbinia/jobs/redis.py

```diff
@@ -35,7 +36,7 @@ class RedisExtractionJob(interface.TurbiniaJob):
   # The types of evidence that this Job will process
   evidence_input = [
       Directory, DockerContainer, GoogleCloudDisk, GoogleCloudDiskRawEmbedded,
-      RawDisk
+      RawDisk, ExpertWitnessFormat
```

Comment:
It seems like there are likely some other jobs that we would want to add this to as well (just based on a quick grep for `RawDisk`), can you double check the other Jobs to make sure?  Thanks!

Reply:
I opt to call it EwfDisk

Reply:
Should be resolved now, does it look ok?